docker-compose.yaml

```diff
@@ -78,7 +78,7 @@ services:
     extends:
       file: docker-compose-lib.yaml
       service: geoportal
-    image: camptocamp/geomapfish-geoportal:2.5
+    image: camptocamp/geomapfish-geoportal:2.6
```

Comment:
Shouldn't it be set to `${DOCKER_TAG}`
Ad we should add `DOCKER_TAG=2.6` in the `env.project` file to also make the build working?

Reply:
I don't know, but I would love having this defined in only one place.

Note that we use camptocamp/geomapfish-tools:latest as base for builder but camptocamp/geomapfish:2.6 as base for runner, this is not really consistent.